src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.figures;
+
+public class IsoscelesTrapezoid extends Figures {
```

Comment:
For future - it's better to have getters and setters as well, so you can change figure state after creation,